fedimint-server/src/consensus/engine.rs

```diff
@@ -457,7 +462,7 @@ impl ConsensusEngine {
 
     #[instrument(target = "fm::consensus", skip(self, item), level = "info")]
     pub async fn process_consensus_item(
-        &self,
+        &mut self,
```

Comment:
`&mut self` is needed on top level functions now to avoid locking/inner-mutability.